app/controllers/users_controller.rb

```diff
@@ -20,10 +28,60 @@ def create
       render :new
     end
   end
+
+  def edit; end
+
+  def update
+    if @user.update_attributes(user_params)
+      flash[:sucess] = t ".profile_update"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @user = User.find params[:id]
+    if @user.destroy
+      flash[:success] = t ".deleted_user"
+    else
+      flash[:danger] = t ".delete_err"
+    end
+    redirect_to users_path
+  rescue ActiveRecord::RecordNotFound
```

Comment:
dry(duplicate code `rescue ActiveRecord::RecordNotFound` at show action and destroy action)